.github/workflows/comment-test-link-merged-pr.yaml

```diff
@@ -0,0 +1,70 @@
+name: Comment GitHub Actions link on a merged PR
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - deployer/**
+      - requirements.txt
+      - .github/actions/setup-deploy/**
+      - "**.yaml"
+
+jobs:
+  # In order to comment on a Pull Request, we need its number. But because we do not
+  # trigger this workflow in a Pull Request context, it is not readily available to us.
+  # Instead, we extract the PR number from the message of the head commit in the push
+  # event payload. In the case of a merged PR, this message will be of the form:
+  # 'Merge pull request #XXX from owner/branch'
+  # This job sets two outputs: continue-workflow (bool, whether or not to run the next
+  # job) and pr-number (str)
+  get-pr-number:
+    runs-on: ubuntu-latest
+    outputs:
+      continue-workflow: ${{ steps.get-pr-number.outputs.continue-workflow }}
+      pr-number: ${{ steps.get-pr-number.outputs.pr-number }}
+    steps:
+      - name: Echo commit message
+        id: get-pr-number
+        shell: python
+        run: |
+          import re
+
+          commit_msg = r"""${{ github.event.head_commit.message }}"""
+          print(f"::set-output name=continue-workflow::{'Merge pull request' in commit_msg}")
+
+          match = re.search('(?<=#)[0-9]*', commit_msg)
+          if match is not None:
+              pr_number = match.group(0)
+              print(f"::set-output name=pr-number::{pr_number}")
+          else:
+              print(f"::set-output name=pr-number::{None}")
+
+  add-pr-comment:
+    # This job depends on the get-pr-number job to have completed successfully and set
+    # the continue-workflow variable to True. It consumes the pr-number variable to
+    # comment on that Pull Request with a link to the deploy-hubs.yaml wiorkflow running
+    # on the master branch.
+    runs-on: ubuntu-latest
+    needs: [get-pr-number]
```

Comment:
```suggestion
    needs: [get-pr-number]
    permissions:
      pull-requests: write
```

Reply:
I _think_ it might actually need to be `issues` because, according to the GitHub REST API, a PR is just a special type of issue. And since we're only commenting, not updating the PR-ness, then I think `issues` is right. For example, we use the `issues` endpoint to create the comment here

https://github.com/2i2c-org/infrastructure/pull/1272/files#diff-e38f6f1c50d8c5fc35bba540c9ba5d99d4a1e817453e5229bda0de776fb1895bR61

Reply:
Ah nice catch!